src/components/RenderError.js

```diff
@@ -24,8 +24,22 @@ import IconTriangleWarning from 'icons/TriangleWarning'
 // SERIOUSLY plz refactor to use <ResetButton>
 import { IconWrapperCircle } from './SettingsPage/ResetButton'
 
-const printError = (error: mixed) => `${String(error)}
-${String((error && error.stack) || 'no stacktrace')}`
+const printError = (error: mixed) => {
+  const print = []
+
+  if (error.message) {
```

Comment:
please assume error could be falsy. `throw null` or shit like that. we never knows.

Reply:
curious that flow didn't complain btw